input_output/file_output.py

```diff
@@ -50,4 +54,40 @@ def save_graph_in_png(filename):
     os.makedirs(os.path.dirname(dot_filename), exist_ok=True)
     os.makedirs(os.path.dirname(png_filename), exist_ok=True)
 
-    subprocess.run(["dot", "-Tpng", dot_filename, "-o", png_filename])
\ No newline at end of file
+    subprocess.run(["dot", "-Tpng", dot_filename, "-o", png_filename])
+
+
+def create_csv_file(name: 'String'):
+    """
+    Creates an empty CSV file with `sep=.` initializer part for Excel compatibility.
+    The filename will be timestamped by prefixing name with the current time:
+    example: YYYYMMDD-HH_MM_SS-name
+    The path of the csv file, including extension .csv, is returned as String.
+    If the folder output_files/csv does not exist it is created.
+    The argument newline = '' to prevens double carriage returns in non-binary mode.
+    :param name: preferred name of the file
+    :return: string csv_filename, path to the file
+    """
+
+    csv_filename = ROOT + '/output_files/csv/' + time.strftime('%Y%m%d-%H_%M_%S') + '-' + name + '.csv'
+    os.makedirs(os.path.dirname(csv_filename), exist_ok=True)
```

Comment:
Ik weet niet waarom deze regel erin staat, maar zonder werkt het ook.

Reply:
De regel zorgt ervoor dat als de map waar de csv naar toe wordt geschreven wordt aangemaakt als deze niet bestaat.

Heb het aangepast in de comments.